src/zcl/definition/cluster.ts

```diff
@@ -44,6 +44,9 @@ const Cluster: {
             dateCode: {ID: 6, type: DataType.charStr},
             powerSource: {ID: 7, type: DataType.enum8},
             appProfileVersion: {ID: 8, type: DataType.enum8},
+
+            legrandUnknown: {ID: 0x00a, type: DataType.octetStr},
```

Comment:
Is this still needed after your second commit?

Reply:
I see why it's needed now, I would like to provide the ID instead (so that there is no need to map it here). Can you provide the herdsman debug log when receiving this message?

Reply:
After more tests it's removable. I was seeing those frames on the official gateway but it's only if i poll them. still haven't found what the purpose is ...